dev-utils/run-script.js

```diff
@@ -51,10 +63,71 @@ function launchSauceConnect() {
   console.log('set MODE=saucelabs to launch sauce connect')
 }
 
+function runE2eTests(configPath) {
+  const webDriverConfig = join(PROJECT_DIR, configPath)
+  runE2eTestsUtils(webDriverConfig, false)
+}
+
+function buildE2eBundles(basePath) {
+  buildE2eBundlesUtils(join(PROJECT_DIR, basePath), err => {
+    err && process.exit(2)
+  })
+}
+
+function runSauceTests(serve = 'true', path = './', ...scripts) {
+  /**
+   * `console.logs` from the tests will be truncated when the process exits
+   * To avoid truncation, we flush the data from stdout before exiting the process
+   */
+  if (process.stdout.isTTY && process.stdout._handle) {
+    process.stdout._handle.setBlocking(true)
+  }
+
+  let servers = []
+  if (serve === 'true') {
+    servers = startTestServers(join(PROJECT_DIR, path))
+  }
+  /**
+   * Decides the saucelabs test status
+   */
+  let exitCode = 0
+  const loggerOpts = {
+    stdout: process.stdout,
+    stderr: process.stderr,
+    parallel: false,
```

Comment:
if you decide to run tasks in serial, let's remove this key and also printLabel, default is false anyways.